src/add-remove.js

```diff
@@ -0,0 +1,14 @@
+import { dataBase, updateValues, updateDatabase } from './functions.js';
+
+export function addTask(task) {
+  dataBase.push(task);
+  updateValues();
+}
+
```

Comment:
- [ ] It would be better if you add the method to delete a single task. As is one of the requirements and helps you to delete a task that you want. 

🛠️ 👍